test/util.test.js

```diff
@@ -18,3 +20,25 @@ describe('Util', () => {
     assert.notStrictEqual(typeOf([]), 'object');
   });
 });
+
+describe('assertString', () => {
+  it('Should throw an error if no argument is provided', () => {
+    assert.throws(() => { assertString(); }, TypeError);
+  });
+
+  it('Should throw an error if the argument is not a string, number', () => {
+    assert.throws(() => { assertString(123); }, TypeError);
+  });
+
+  it('Should throw an error if the argument is not a string, Object', () => {
+    assert.throws(() => { assertString({}); }, TypeError);
+  });
+
+  it('Should throw an error if the argument is not a string, Array', () => {
+    assert.throws(() => { assertString([]); }, TypeError);
+  });
+
+  it('Should not throw an error if the argument is a string', () => {
+    assert.doesNotThrow(() => { assertString('testing'); });
```

Comment:
Can we keep the empty string unit test? Then we will not forget that we accept that on purpose

Reply:
Yes, we can. I'll create and commit it.

Reply:
Now we've this unit teste:
![image](https://github.com/validatorjs/validator.js/assets/68362954/64e03b29-9f24-4304-93e0-14fb76fc185f)

Reply:
while we are at it, we should likely also test cases for `null` and `undefined`.

Maybe `NaN` too? And I guess `true` and `false` should also be tested against, for full coverage of all types :-)

Reply:
I created the `null` input validation and also added the suggested tests. I tried to add the following test:

`  
  it('Should not throw an error if the argument is a String instance', () => {
      assert.doesNotThrow(() => { assertString(new String('antidisestablishmentarianism')); });
    });
`
But some ESLint rules prevent this approach, ESLint "no-new-wrappers.